Makefile

```diff
@@ -21,6 +21,11 @@ dry-core:
 core:
 	applyconfig --config-dir core-services --confirm=true
 
+# these are useful for devs
+jobs:
+	docker pull registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest
```

Comment:
maybe we should start suggesting podman? :)

Reply:
if you want to make a follow-up that determines which cri you have and uses that ...